docs/layers/accounts/deploy-accounts.mdx

```diff
@@ -94,6 +93,46 @@ This step-by-step process outlines how to deploy AWS accounts using `atmos` work
     member accounts in the Organization.
   </Step>
 
+  <Step>
+    ## <StepNumber/> Confirm Root Account Name
+
+    After deploying the AWS Organization, you need to verify the root account name and configure it in the `account-map` catalog, if not already set. This is a critical step that prevents errors in subsequent deployments.
+
+    **Why this is needed:** The root account has two different names:
+    <Steps>
+    - **AWS Account Name**: The name you typed when originally creating the AWS account (often "root" but can be different)
+    - **Account Alias**: The deterministic name set in the account configuration (e.g., "core-root")
+    </Steps>
+
+    The account-map component needs to know the actual AWS account name to properly map accounts.
+
+    **To find your root account name:**
+
+    ```bash
+    # Make sure you're using a user with access to your root account
```

Comment:
Will they have acccess to the superadmin at this point

Reply:
yes they will. added suggestion to use superadmin specifically